detection_rules/schemas/__init__.py

```diff
@@ -256,6 +256,12 @@ def migrate_to_8_9(version: Version, api_contents: dict) -> dict:
     return strip_additional_properties(version, api_contents)
 
 
+@migrate("8.10")
+def migrate_to_8_10(version: Version, api_contents: dict) -> dict:
+    """Default migration for 8.10."""
+    return strip_additional_properties(version, api_contents)
+
+
```

Comment:
When do we add 8.11?

Reply:
When we turn `main` into `8.12`. This PR will make main `8.11` and create a `8.10` branch.